outputs.tf

```diff
@@ -16,20 +16,21 @@
 
 output "addresses" {
   description = "List of address values managed by this module (e.g. [\"1.2.3.4\"])"
-  value       = ["${local.ip_addresses}"]
+  value       = local.ip_addresses
```

Comment:
do we change type from list [] to string here? this is public interface change, don't sure it's fine

Reply:
I mean some users might already use this outputs as list

Reply:
It is list by default because of :
  ip_addresses = concat(
    google_compute_address.ip.*.address,
    google_compute_global_address.global_ip.*.address,
  )

Reply:
you right